demo/src/main/java/com/novoda/demo/LandingActivity.java

```diff
@@ -23,6 +23,7 @@ protected void onCreate(Bundle savedInstanceState) {
         final EditText mpdAddress = findViewById(R.id.mpd_address);
         final EditText licenseAddress = findViewById(R.id.license_address);
         final CheckBox downloadLicense = findViewById(R.id.download_license);
+        final CheckBox useContentProtection = findViewById(R.id.use_content_proteciton);
```

Comment:
What's this stuff about `contentProtection` and how does it relates to user defined caption style?